tools/odbc/test/tests/alter.cpp

```diff
@@ -0,0 +1,51 @@
+#include "../common.h"
+
+using namespace odbc_test;
+
+TEST_CASE("Alter", "[odbc]") {
```

Comment:
Nit: Maybe elaborate a bit more on the test names, e.g., `"Test ALTER TABLE statement"`